src/passes/pass.cpp

```diff
@@ -204,15 +204,18 @@ static void dumpWast(Name name, Module* wasm) {
 
 void PassRunner::run() {
   static const int passDebug = getPassDebug();
-  if (!isNested && (options.debug || passDebug)) {
+  static const bool printInfo = getPassDebug() >= 2;
```

Comment:
This part is probably my least favorite part of the change, because we're overloading the environment variable to mean multiple things:

1. Validate the IR between passes.
2. Dump out intermediate IR between passes.
3. Saving the IR from the last pass for debugging purposes.
4. Write timing information.
5. Status updates about how things are progressing.

It's not clear to me that all of these should really be crammed into a single option; it feels cleaner to have multiple command-line options, say, `--print-times`, `--validate-between-passes`, `--dump-pass-ir`, or similar.  Then we would ditch the environment variable altogether.  WDYT?

Reply:
Yeah, if there's a reasonable way to do this with options (instead of an env var) that would be cleaner. The thing is that then we need to add those options to every tool, which is hard to do nicely. `optimization-options.h` is the closest we have, I guess.